src/enums/ConsellingRelation.ts

```diff
@@ -1,4 +1,5 @@
 export enum CounsellingRelation {
```

Comment:
Could you rename the file to "CounsellingRelation.ts"?
I know that typo existed before you made the changes but anyways 😅

Reply:
Done!